inference-engine/src/transformations/include/transformations/op_conversions/low_latency.hpp

```diff
@@ -0,0 +1,5 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/pass/low_latency.hpp>
```

Comment:
Look like it's more relevant to common_optimizations

Reply:
ok, will move it

Reply:
moved